### DISCUSSION TITLE
radial gradient in pseudo elements does not compiled

### DISCUSSION THREAD

**Discussion by JUSTIVE** [tailwindlabs/tailwindcss#13272] [Help] [answered]
**What version of Tailwind CSS are you using?**

3.4.1

**What build tool (or framework if it abstracts the build tool) are you using?**

"postcss": "8.4.24",
"nextjs": "14.0.4"

**What version of Node.js are you using?**

v20.10.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

https://play.tailwindcss.com/TT1c2xZwfW

**Describe your issue**

`radial-gradient` in `bg-` with `after:` prefix does not generate equivalent CSS. if I apply same style(radial-gradient(#EBEEF3 0%, #EBEEF3 67%, white 67%)) directly into pseudo-element in devtools, it works.

**Comment by adamwathan** [comment, marked as answer]
Hey! Class names can't have spaces in them, you need to use underscores for spaces in arbitrary values:

https://play.tailwindcss.com/251jGsgl1C

**Reply by JUSTIVE** [reply]
OMG. what a shame! thanks for the answer!
